include/swift/AST/ActorIsolation.h

```diff
@@ -180,6 +180,12 @@ class ActorIsolation {
     return parameterIndex;
   }
 
+  /// Returns true if this actor-instance isolation appllies to the self
```

Comment:
typo: `appllies` -> `applies`

Reply:
I'll fix that on main. This is for a cherry-pick to 6.0.